malasakit-django/pcari/models.py

```diff
@@ -513,6 +513,34 @@ class Meta:
         unique_together = ('respondent', 'question')
 
 
+class Location(models.Model):
+    """
+    A ``Location`` represents a named government-designated area in the world.
+
+    Attributes:
+        country (str): The name of the country of the location.
+        province (str): The name of the province (in the United States, this
+            would be analogous to a state).
+        municipality (str): The name of a municipality (can vary from a county
+            to a city or town).
+        division (str): The name of the smallest possible government unit
```

Comment:
should we also mention Barangay somewhere for the PH team

Reply:
I don't think this should be part of the toolkit documentation, as the toolkit should be general-purpose. Since a barangay is an atomic administrative division, it should be clear that this is the barangay field (can also look at all the values in this column, which may have "Barangay" in their names).